internal/testcli/runner.go

```diff
@@ -39,6 +39,8 @@ type Runner struct {
 	StderrLines <-chan string
 
 	errch <-chan error
+
+	NoLog bool
```

Comment:
Can you flip this around and name it "LogOutput"? It can initialize to true from `NewRunner`.

It avoids the double negation in conditionals.

Reply:
Good idea, changed to Verbose.